packages/contracts-bedrock/contracts/L1/L1StandardBridge.sol

```diff
@@ -100,6 +100,21 @@ contract L1StandardBridge is StandardBridge, Semver {
         StandardBridge(_messenger, payable(Predeploys.L2_STANDARD_BRIDGE))
     {}
 
+    /**
+     * @notice Allows EOAs to deposit ETH by sending directly to the bridge.
+     */
+    receive() external payable override onlyEOA {
+        // Notice this emits an `ETHDepositInitiated` event.
+        // Whereas `StandardBridge` contract's `receive` fallback will only call
+        // `_initiateBridgeEth`, which will emit an `ETHBridgeInitiated` event.
```

Comment:
I think this comment should move into the natspec, and then have a separate comment explaining what the assembly is doing/why it's needed.

```suggestion
    /**
     * @notice Allows EOAs to deposit ETH by sending directly to the bridge.
     *
     * 		   Emits both an `ETHDepositInitiated` and `ETHBridgeInitiated` event.
     */
    receive() external payable override onlyEOA {
    	// Explain b and the assembly...
```